src/main.rs

```diff
@@ -45,6 +45,19 @@ fn run() -> Result<(), failure::Error> {
         .author("ashley g williams <ashley666ashley@gmail.com>")
         .setting(AppSettings::ArgRequiredElseHelp)
         .setting(AppSettings::DeriveDisplayOrder)
+        .subcommand(
+            SubCommand::with_name("kv")
+                .about(&*format!(
+                    "{} Interact with your Workers KV Store",
+                    emoji::KV
+                ))
+                .subcommand(
+                    SubCommand::with_name("add")
```

Comment:
we decided to go with "create", didn't we?

Reply:
yes, but that was as of this morning. new commit incoming soon.